projects/ngneat/reactive-forms/src/lib/utils.ts

```diff
@@ -44,3 +44,16 @@ export function removeError<E>(errors: E, key: keyof E) {
   delete updatedErrors[key];
   return Object.keys(updatedErrors).length > 0 ? updatedErrors : null;
 }
+
+export function isObjectEmpty(obj) {
+  return Object.keys(obj).length === 0 && obj.constructor === Object;
+}
+
+/**
+ * @return {boolean} True if arrays are identical.
+ */
+export function compareArraysContent(left, right): boolean {
+  left = Array.isArray(left) ? left : [];
+  right = Array.isArray(right) ? right : [];
+  return left.length === right.length && left.every(value => right.includes(value));
```

Comment:
Why not using `some` instead of `every`?

Reply:
Idea behind that function is that it would return `true` if arrays are identical.
Ex. `compareArraysContent([ 1, 2 ] , [ 1, 2, 3 ]);`

When compared with `every` it would yield  false, since `every` would compare all elements of array.
When compared with `some` it would yield  true.